src/tools/scripts/archived/remove-user-structure-inactif.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+'use strict';
+
+const { execute } = require('../../utils');
+
+execute(__filename, async ({ db, logger, exit, Sentry }) => {
+  try {
+    // Recupération des structures dont le statut est en 'ABANDON' ou 'ANNULEE'.
+    const structures = await db.collection('structures').find({ $or: [{ statut: 'ABANDON' }, { statut: 'ANNULEE' }] }).toArray();
```

Comment:
$in  'EXAMEN_COMPLEMENTAIRE_COSELEC' , 'REFUS_COSELEC' 'ABANDON' , 'ANNULEE' ,  DOUBLON' , 'NEGATIF'